app/components/Settings/SettingsPage.tsx

```diff
@@ -73,19 +75,33 @@ const settingsInputs: readonly SettingsInput[] = [
     type: SettingType.SwitchList,
     isAdvanced: true,
   },
+  {
+    key: 'showAlphaSettings',
+    type: SettingType.Switch,
+  },
+  {
+    key: 'alphaPath',
+    type: SettingType.Path,
+    isAlpha: true,
```

Comment:
Pressing "Find" and then cancelling in the popup for some reason makes "showa dvanced settings" toggle back to false. It also makes the "Save" button show up even though the input is empty (might be an old manager bug). Same thing happens with the OWML path input too now.

Reply:
It seems to reset everything back to default settings.